### ISSUE TITLE
On Windows, path with spaces are not double quoted.

### ISSUE THREAD

**Issue by blemelin** [Byron/open-rs#53]
Let's say we want to open this file with Visual Studio Code : `D:\projects\work 5.txt`.

```rust
open::with("D:\\projects\\work 5.txt", "code").unwrap();
```

Doing this ends up opening two files : `work` and `5.txt`. For this to work properly, the path must be inside double quotes like this : 

```rust
open::with("\"D:\\projects\\work 5.txt\"", "code").unwrap();
```

Is this the intended behaviour ? On other platforms, this crate seems to handle spaces perfectly fine, so I created a fix for Windows in a fork of this repository. I can open a Pull Request if you are interested. A lot of programs can open multiple files at the same time, but this crate seem to be intended to open one file at a time, hence this issue.

**Comment by Byron** [comment]
Thanks for letting me know, and a fix for this would be greatly appreciated.

This definitely isn't intended behaviour and I believe the difference in handling of arguments might be in the nature of windows programs, maybe. On linux, an argument passed to a program is just that, and it can contain any character including whitespace, no additional splitting is performed as it doesn't go through a shell. On windows, a single string is passed as concatenation of all arguments, as I understood it, which then is split by the program itself. Hence one will always have to escape certain characters to avoid the splitting from happening, which isn't done yet.

**Comment by blemelin** [comment]
I've created the requested pull request. See PR #54. Thanks a lot for your work. :smiley:

**Comment by Byron** [comment]
Thank you!
The fix is now available in [v3.0.3](https://github.com/Byron/open-rs/releases/tag/v3.0.3).
